src/lib/containers/table/SynapseTable.tsx

```diff
@@ -279,10 +265,6 @@ export default class SynapseTable extends React.Component<
         },
       )
       try {
-        // initialize mapEntityIdToHeader
-        referenceList.forEach(el => {
-          mapEntityIdToHeader[el.targetId] = EMPTY_HEADER
-        })
```

Comment:
don't initialize each entity with an empty header. if we don't get a header, then it will be `undefined` in the map.